tests/func/test_data_cloud.py

```diff
@@ -149,6 +167,18 @@ def get_ssh_url():
     )
 
 
+def get_ssh_url_mocked(user, port):
+    if os.name == "nt":
+        url = "ssh://{}@127.0.0.1:{}:{}".format(
+            user, port, get_local_storagepath(directory=here)
```

Comment:
I don't think it should create dirs in `here`. Why `get_local_storagepath` doesn't work?

Reply:
Cool, you are right.